hkgraph.js

```diff
@@ -80,29 +81,35 @@ HKGraph.prototype.setEntity = function(entity)
 	// Update parent
 
 	// Clean old entity
-	let oldParent = this.getEntity(oldEntity.parent);
-	if(oldParent)
+	if(oldEntity.parent) 
```

Comment:
Can we ensure that ```oldEntity``` **always** has the ```parent``` property? Without ensuring this, this change may causes property not found-like errors. The old-way, i.e., `let oldParent = this.getEntity(oldEntity.parent); if(oldParent)` is able to check whenever ```oldEntity``` indeed has the ```parent``` property.

Reply:
Thanks @vitornl. I will include a verification using **hasOwnProperty**, although in my recent tests I've noticed that JS is able to check the value of an inexistent property without errors. Here follows an example:
![image](https://user-images.githubusercontent.com/4651610/83534529-5a616180-a4c7-11ea-8076-627b35964220.png)